src/backend/backend.py

```diff
@@ -143,3 +175,28 @@ def activate_room_debug(room_hash):
     else:
         return jsonify(success="deactivated")
 
+@app.route("/otp/list")
+@auth.login_required
+def list_otps():
+    otps = OTP.select()
+    otp_list = [(o.otp, o.label) for o in otps if not o.used]
+    return str(otp_list)
+
+
+@app.route("/otp/create")
+@auth.login_required
+def create_otp():
+    label = request.args.get("label","")
+    new_otp = OTP.create(label=label)
+    new_otp.save()
+    return str(new_otp)
```

Comment:
What happens here? It this like to JSON?

Reply:
For the record: This is for debugging only. In the future we need to make this JSON